unit-tests/src/test/scala/scala/scalanative/regex/ExecTest.scala

```diff
@@ -463,7 +451,7 @@ class ExecTest {
 
         case 'C' | 'N' | 'T' | '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' |
             '8' | '9' =>
-          System.err.println("skip: %s\n".format(line))
+          //System.err.println("skip: %s\n".format(line))
```

Comment:
Consider removing the line instead. It was probably left there by mistake.

Reply:
I found another one that was outputting but I think it might be helpful for someone that has to work on this code again. The other one would leave an empty `if` - I could put a comment above `// left for debugging` or something. WDYT?